### DISCUSSION TITLE
ps_root_dispatch: No such file or directory

### DISCUSSION THREAD

**Discussion by ArthurPeabody** [NetworkConfiguration/dhcpcd#419] [Q&A]
I'm trying to make a wireless connection between my computer and new Xfinity router. ifconfig and iwconfig work, but dhcpcd returns 'ps_root_dispatch: No such file or directory'

**Comment by rsmarples** [comment]
How are you starting dhcpcd? To me, it sounds like an issue using the udev plugin - do you use udev?

**Comment by flotho** [comment]
Hi @ArthurPeabody , did you solve your issue ?

**Reply by ArthurPeabody** [reply]
I don't know.   The error stopped 2025 January 28.  I don't know whether I did something to address the problem or it disappeared when I started connecting wiredly.  I'd like to know what the error means in any case.  I do use udev.  I don't know why I failed to reply to Mr Marples.
